app/models/user.rb

```diff
@@ -77,6 +78,10 @@ def print_id
     %(#{print} - #{id})
   end
 
+  def print_email
+    %(#{print} <#{email}>)
+  end
+
```

Comment:
Moving `print` to `to_s` and removing full_name.

Removed the use of `print` and `full_name`, the `print_email` is useful for proper format of email 
and `print_id` is used as a label_method in `simple_form` which is quite useful.

Great comments :)